zerver/webhooks/dialogflow/fixtures/alternate_result.json

```diff
@@ -34,7 +34,7 @@
   "alternateResult": {
     "fulfillment": {
       "speech": "Weather in New Delhi is nice!"
-    },
```

Comment:
Dialogflow isn’t really going to send us invalid JSON…right?

Reply:
Our fixtures were largely collected from actually using the API of third-party products.  I don't know for this one specifically, but ... I'm not sure we can rule out the possibility of invalid JSON on these webhook APIs.

Reply:
https://www.javaroad.cn/questions/76340 shows a sample payload with an `alternateResult` with many more fields—looks like the same format as `result`. So I’m fairly convinced this fixture was hand-edited.